packages/messaging/src/listeners/sw-listeners.ts

```diff
@@ -207,11 +207,29 @@ async function getWindowClient(url: URL): Promise<WindowClient | null> {
   return null;
 }
 
+/**
+ * @return If runtime is chrome extension manifest v3.
```

Comment:
Should this be something more like, "If runtime is Chrome, use extension manifest v3?"

I feel like it needs a verb (especially if this is a public string parsed into reference docs).

Reply:
sg. updated.